src/Aspire.Hosting/Dcp/DcpOptions.cs

```diff
@@ -113,16 +111,22 @@ public void Configure(DcpOptions options)
         var dcpPublisherConfiguration = configuration.GetSection(DcpPublisher);
         var assemblyMetadata = appOptions.Assembly?.GetCustomAttributes<AssemblyMetadataAttribute>();
 
-        if (!string.IsNullOrEmpty(dcpPublisherConfiguration[nameof(options.CliPath)]))
+        // Find dcp tools path in the following order:
+        // 1. Environment variable ASPIRE_DCP_DIR
+        // 2. Configuration value
+        // 3. Assembly metadata
+        if (Environment.GetEnvironmentVariable("ASPIRE_DCP_DIR") is var dcpDirEnvVar && !string.IsNullOrEmpty(dcpDirEnvVar))
```

Comment:
Should this be `DOTNET_ASPIRE_DCP_DIR`?

Reply:
I'd think so, this is the prefix we have been using for environment variables in other places, including this file